wallet/src/de/schildbach/wallet/ui/AboutActivity.kt

```diff
@@ -84,7 +84,7 @@ class AboutActivity : BaseMenuActivity(), SensorEventListener {
 
     private fun showFirebaseInstallationId() {
         FirebaseInstallations.getInstance().id.addOnCompleteListener { task ->
-            firebase_installation_id.isVisible = task.isSuccessful && BuildConfig.DEBUG
+            firebase_installation_id.isVisible = task.isSuccessful
```

Comment:
Do we want to show the firebase installation id in production?

Reply:
Yes, it was requested to make it easier to test In-App-Messaging